### ISSUE TITLE
Create hidden text in nodes with long text. A good and practical idea.

### ISSUE THREAD

**Issue by aaa1386** [freeplane/freeplane#1795]
Hello.
I propose a good idea.
Sometimes the text we write inside a node is a long text.
Fortunately, the maximum width of the node can be set horizontally.
See the figure below.

![image](https://github.com/freeplane/freeplane/assets/38586259/8e191eb5-07ad-4d78-9a56-8c88afcca379)

  The only problem is that it lengthens the node vertically and there is no adjustment for this.
One way is to transfer the text of the node in the  "Note" section and write a title for that long text in the node.
 
But maybe some users don't want to use "Note".
There is another very good solution. pay attention:

I suggest: create an option in the node style editing section: (maximum vertical length)
which has two modes:

1- The vertical length can be adjusted by any desired number.
  Yes, it is true that some of the long text remains hidden. But this is a very good thing. Because it can be set that some of the text that is displayed is the title for the long text and the hidden text is the main body.
Whenever the node is selected, all the text will be automatically displayed, and when the node is disabled, only the amount of text that is set with the option (maximum vertical length) will be displayed.

  We can also create a style for the specific setting we are doing.

2- The mode in which the number is not set will work as before and the node will be created with a long height.

If the above idea is implemented, to a large extent, we will not need to write long texts in the "note" section.

Another suggestion is that when a node has hidden text, an icon and a symbol will be displayed on that node, which shows that the node has hidden text.

How attractive. Managing the length and width of all nodes and having long texts at the same time.

**Comment by euu2021** [comment]
Yes, in my daily use of FP, limiting the size of nodes vertically is essential, as I explained at https://github.com/freeplane/freeplane/discussions/401#discussioncomment-2845218. 

It can be done with scripts, although it's not perfect, yet. See the discussion #1293 and linked threads.

**Comment by aaa1386** [comment]
Yes, it is true that it is possible to hide long texts even in Freeplane software.
This approach has a weakness that I will explain in due course.

**Comment by aaa1386** [comment]
By clicking on the node icon that has hidden text. Hidden text is not displayed.

See the video below.


https://github.com/freeplane/freeplane/assets/38586259/eb7b0bfa-736c-4fb2-98d6-0f3362a70eb6
  "View->Minimize Nodes" either puts the node in hidden text mode or takes it out of it.
It is suggested to make it possible in the software to display the hidden text by clicking on the triangle icon and to hide the text by clicking on the icon again.
